tinyagentos/__main__.py

```
@@ -0,0 +1,62 @@
+    import uvicorn
+
+    env_host = os.environ.get("TAOS_HOST")
+    env_port = os.environ.get("TAOS_PORT")
+    env_data_dir = os.environ.get("TAOS_DATA_DIR")
+
+    if env_host or env_port:
+        host = env_host or "127.0.0.1"
+        port = int(env_port) if env_port else 6969
+    else:
+        config = load_config(PROJECT_DIR / "data" / "config.yaml")
+        host = config.server.get("host", "0.0.0.0")
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)